Update/toki_009.txt

```diff
@@ -3077,7 +3077,7 @@ void main()
 			NULL, "\"A male student was murdered and the culprit committed suicide.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 0, "ps3/s11/00/vt3b_honjo2147", 256, TRUE);
 	OutputLine(NULL, "…それだげなら、ただの享楽殺人か猟奇殺人だ』",
-			NULL, " ...If that's the case, it could simply be a case of committing murder for personal enjoyment or to sate curiosity.\"", Line_Normal);
+			NULL, " ...If that's was all there was to it, it could simply be a case of a murder committed for personal enjoyment or to sate curiosity.\"", Line_Normal);
```

Comment:
```suggestion
			NULL, " ...If that was all there was to it, it could simply be a case of a murder committed for personal enjoyment or to sate curiosity.\"", Line_Normal);
```